primedev/squirrel/squirrel.cpp

```diff
@@ -620,32 +620,25 @@ template <ScriptContext context> void ConCommand_script(const CCommand& args)
 }
 
 // literal class type that wraps a constant expression string
-template <size_t N> struct TemplateStringLiteral
+template <ScriptContext context> SQRESULT SQ_StubbedFunc(HSquirrelVM* sqvm)
 {
-	constexpr TemplateStringLiteral(const char (&str)[N])
-	{
-		std::copy_n(str, N, value);
-	}
+	SQStackInfos si;
+	g_pSquirrel<context>->sq_stackinfos(sqvm, 0, si);
 
-	char value[N];
-};
-
-template <ScriptContext context, TemplateStringLiteral funcName> SQRESULT SQ_StubbedFunc(HSquirrelVM* sqvm)
-{
-	spdlog::info("Blocking call to stubbed function {} in {}", funcName.value, GetContextName(context));
+	spdlog::warn("Blocking call to stubbed function {} in {}", si._name, GetContextName(context));
```

Comment:
Just noticed that previously we would print as `info` instead of `warn` which tbh was kinda stupid, so good idea on changing that in the process ^^